packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -264,6 +277,6 @@ export class PureDataObjectFactory<TObj extends PureDataObject<I>, I extends Dat
 
         await context.attachRuntime(this, runtime);
 
-        return instance;
+        return runtime.entryPoint?.get() as unknown as TObj;
```

Comment:
This ends up changing existing behavior by initializing exising data objects, i.e., this ends up calling `finishInitialization` on the data object. By default, data objects avoid initializing existing data object because we don't want to initialize user code in the summarizer client.